bin/shundle

```diff
@@ -139,9 +139,9 @@ _shundle_install()
         _shundle_install_github__revision=":${1##*:}"
         _shundle_install_github__bundle="${1%$_shundle_install_github__revision}"
 
-        if [ "${GIT_VERSION}" -lt "230" ]; then #non interactive git
+        #if [ "${GIT_VERSION}" -lt "230" ]; then #non interactive git
```

Comment:
What was the reasoning behind this?, with the new GIT_VERSION logic this variable should be always populated, right?, I think this if statement should not be commented.

Reply:
Yes, you're right, It was my fault, my apologies 🙏